DOJO-QUICKSTART.md

```diff
@@ -0,0 +1,43 @@
+# dojo quickstart guide
+
+## Setup
+
+### Objectives: 
+- Give a professional assessment in a straight forward, modern, presentation format
+- Efficiently record & report on assessment results
+- TODO: Persistantly store results in one place
+ - while not a current function this will be a future need  
+
+### Assumptions: 
+- Running from MacBook
+
+### Pre-reqs: 
+- Nodejs (Tested with Node 6.2)
+- git
+- Chrome 
+
+### Let's get started on the cli.
+- Download & Install [node](https://nodejs.org/dist/v6.2.1/node-v6.2.1.pkg) 
```

Comment:
Could we recommend one method or the other?   
Also, would it be better to point to a non version-specific download link, in case paths change and whatnot?